content/en/pod.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Pod
+status: Feedback Appreciated
+category: concept
+tags: ["infrastructure", "fundamental", ""]
+---
+
+## What it is
+
+In [cloud native computing](/cloud-computing/), a "pod" is a basic building block for deploying and managing containerized applications. 
+A pod represents a single instance of an application, and can contain one or more [containers](/container/).
+[Kubernetes](/kubernetes/) manages pods as part of a larger deployment, and can scale pods up or down depending on the application's resource requirements.
+
+## Problem it addresses
+
+Containers provide a lightweight and efficient way to package and deploy applications, but they can't do everything on their own. 
+Applications often require additional resources and services, such as storage or networking, which need to be managed and configured separately.
+
+This is where pods come in. A pod provides a way to group containers together and manage them as a single unit. 
+This makes it easier to ensure that all the resources and services in application needs are available and configured correctly.
+
+## How it helps
+
+Isolation: Each pod has its own isolated environment, including its own network stack and file system. This ensures that containers in the same pod can't interfere with each other, and helps maintain security and stability.
+
+Resource sharing: Containers in the same pod can share resources, such as CPU and memory, which allows for efficient use of resources.
+
+Simplified management: Pods make it easier to manage groups of containers as a single unit, rather than managing each container individually. 
```

Comment:
I think pods allow us to manage a group of tightly related containers as a unit. For instance, a sidecar container must be next to the "regular" container. To function correctly, they need to be together. Same thing for any other resource that is shipped in a pod. While we want them to be isolated, they can't be separated.

Reply:
Agreed